lib/keepr/groups_creator.rb

```diff
@@ -1,10 +1,12 @@
 # frozen_string_literal: true
 
 class Keepr::GroupsCreator
-  def initialize(target)
+  def initialize(target, language)
```

Comment:
To maintain backwards compatibility, I prefer a to use this:

`def initialize(target, language = :de)`

Reply:
Good point, I thought about it after open PR 👌